sql/src/test/java/io/crate/analyze/RepositoryParamValidatorTest.java

```diff
@@ -91,6 +92,14 @@ public void testHdfsSecurityPrincipal() throws Exception {
         assertThat(settings.get("uri"), is("hdfs://ha-name:8020"));
     }
 
+    @Test
+    public void testHdfsUriDefaultValueIsNull() throws Exception {
+        GenericProperties genericProperties = new GenericProperties();
+        genericProperties.add(new GenericProperty("path", new StringLiteral("/user/name/data")));
```

Comment:
`GenericProperties.EMPTY` can be used instead (simplify test case)

Reply:
removed all hdfs-related changes for now - thus this is not relevant anymore.